packages/generator-single-spa/src/react/templates/package.json

```diff
@@ -3,7 +3,7 @@
     "start": "webpack-dev-server",
     "build": "webpack --mode=production",
     "analyze": "webpack --mode=production --env.analyze=true",
-    "lint": "eslint src",
+    "lint": "eslint src --ext js<% if (typescript) { %>,ts,tsx<% } %>",
```

Comment:
Dumb queston: Any reason why we couldn't just have all three here always? It wouldn't find `ts` or `tsx` unless you created them.

Reply:
Responded in https://github.com/single-spa/create-single-spa/pull/62#discussion_r405682730